FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+std_msgs::Int8 percent_full;
+ros::Publisher pub_percent_full("percent_full", &percent_full);
+ros::NodeHandle nh;
+
+void setup()
+{
+  nh.initNode();
+  Serial.begin(9600);
+  nh.getHardware()->setBaud(9600);
+  nh.advertise(pub_percent_full);
+}
+
+long publisher_timer;
+const int READ_PIN = A7;
+
+void loop() {
+  if (millis() > publisher_timer) {
+    // outputs range from ~494 at empty to ~714 at full
+    // so shift that to 0-220 and scale to 0-100
+    percent_full.data = ((analogRead(READ_PIN) - 494) * 100 / 220);
+    // then limit it on each end to enforce bounds
+    percent_full.data = max(min(percent_full.data, 100), 0);
+    publisher_timer += 200;
+    pub_percent_full.publish(&percent_full);
+  }
+  nh.spinOnce();
+}
+
+int max(int v1, int v2) {
```

Comment:
for max and min. We don't typically like to redefine really basic functions. 

If you want to include the Arduino.h file they have this exact function builtin 
https://www.arduino.cc/reference/en/language/functions/math/min/

Alternatively you can use std::min and std::max

https://en.cppreference.com/w/cpp/algorithm/min

Reply:
I think those are in here because I didn't immediately find where the builtin was, and found this easier. Thanks for the fix!